Library/Basic Set/Basic Set Advantages.adq

```diff
@@ -30512,6 +30512,13 @@
 				}
 			],
 			"base_points": -15,
+			"features": [
+				{
+					"type": "reaction_bonus",
+					"situation": "from others who realise you are a weirdness magnet (except parapsychologists, cultists, conspiracy theorists, thrill-seekers)",
```

Comment:
Please use American spelling ("realize" instead of "realise"). Thanks.

Reply:
Ah yes, thanks. Was trying to remember to spell things wrong 😁 but missed that one. Will update tonight.